packages/houdini-utils/src/index.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('init', () => {
```

Comment:
this shouldn't be necessary for package scaffold - addressed https://github.com/microsoft/fluentui-contrib/pull/121

Reply:
cool waiting for the merge :)